datamodel_code_generator/format.py

```diff
@@ -12,8 +12,6 @@
 
 
 class PythonVersion(Enum):
-    PY_36 = '3.6'
-    PY_37 = '3.7'
```

Comment:
@jorenham 
I agree with you it's to drop support 3.7 from runtime support list.
Should we delete PY_37 for output?
Personally, I'm happy to drop the old version from the output model list.
But I want to think about it again. :thinking: